editor/editor_node.cpp

```diff
@@ -1417,7 +1417,7 @@ void EditorNode::_display_top_editors(bool p_display) {
 }
 
 void EditorNode::_set_top_editors(Vector<EditorPlugin *> p_editor_plugins_over) {
-	editor_plugins_over->set_plugins_list(p_editor_plugins_over);
+    editor_plugins_over->set_plugins_list(p_editor_plugins_over);
```

Comment:
Yeah qtcreator have spaces by default, I was correcting them manually later but I missed quite a lot of lines